src/main.rs

```diff
@@ -16,10 +17,15 @@ mod protocols;
 mod state;
 mod utils;
 
-static POSSIBLE_BACKENDS: &[&str] = &[
-    "--winit : Run magma as a X11 or Wayland client using winit.",
-    "--tty-udev : Run magma as a tty udev client (requires root if without logind).",
-];
+#[derive(Parser, Debug)]
+#[command(version, about, long_about = None)]
```

Comment:
the previous implementation had some help context, which isnt there anymore. should probably add it

Reply:
Oops, you're right

Reply:
Fixed